cli/run_parser.py

```diff
@@ -146,7 +146,7 @@ def main(
     tasks = []
     counter = 0
     for path in files_to_parse:
-        if TEST_RUN and counter > 100:
+        if bool(TEST_RUN) and counter > 10:
```

Comment:
the pattern i've been using elsewhere is to define a `str_to_bool` function `str_to_bool = lambda var: var.lower() == "true"` - that way env variables are case-insensitive which is more predictable for a developer

Reply:
Ahh nice I'll integrate that!